packages/nextjs-plugin/src/get-hashicorp-packages.test.ts

```diff
@@ -19,4 +19,16 @@ describe('getHashicorpPackages', () => {
       ]
     `)
   })
+
+  test('includes packages from package.json dependencies', () => {
+    expect(
+      getHashicorpPackages(
+        path.join(path.dirname(currentFilePath), '__fixtures__', 'package-json')
```

Comment:
This `package-json` fixture dir is an arbitrary name, right? (This particular name was a bit confusing at first)

Reply:
Yep, just arbitrary. Why was it confusing?

Reply:
I thought it could've been a typo for _`package.json`_. Took me a second to realize it was a fixture directory

Maybe just a _me_ problem 😅

Reply:
Fair 😄